libs/usb-mocking/src/shared.ts

```diff
@@ -0,0 +1,56 @@
+import { $ } from 'zx';
+import { join } from 'path';
+
+export const MOCK_IMAGE_PATH = join(process.cwd(), 'usb-mock.img');
```

Comment:
Is this a safe path if calling these scripts from another project, as we will in the Dev Dock?